recipes/kdtree/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "kdtree" %}
+{% set version = "0.16" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 386df6c7816a05e0fab974e3035df944f99ef68b5615f4a416771391e33d7534
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - kdtree
+
+about:
+  home: https://github.com/stefankoegl/kdtree
+  license: ISC (ISCL)
+  license_family: OTHER
+  license_file: LICENSE
+  summary: A Python implementation of a kd-tree
+  doc_url: https://python-kdtree.readthedocs.org/
+  dev_url: https://github.com/stefankoegl/kdtree
+
+extra:
+  recipe-maintainers:
+    - lincrosenbach
```

Comment:
@lincrosenbach could you please add me as a maintainer of this feedstock when it gets created? Thanks!

Reply:
Sure!